doc/source/user_guide/indexing.rst

```diff
@@ -1158,6 +1158,39 @@ Mask
    s.mask(s >= 0)
    df.mask(df >= 0)
 
+.. _indexing.np_where:
+
+Setting with enlargement conditionally using :func:`numpy`
+----------------------------------------------------------
+
+An alternative to :meth:`~pandas.DataFrame.where` is to use :func:`numpy.where`.
+Combined with setting a new column, you can use it to enlarge a dataframe where the
+values are determined conditionally.
+
+When you have two choices to choose from, say in the following dataframe, set the color
+to 'green' when the second column has 'Z'.  You can do the following:
+
+.. ipython:: python
+
+   df = pd.DataFrame({'col1': list('ABBC'), 'col2': list('ZZXY')})
+   df['color'] = np.where(df['col2'] == 'Z', 'green', 'red')
+   df
+
+If you have more than one conditions, you can use :func:`numpy.select` to achieve that.
```

Comment:
``more that one condition`` (singular)
Or maybe replace it with ``multiple conditions``?

Reply:
done